three60.js

```diff
@@ -1,48 +1,48 @@
 function three60() {
+
+	"use strict";
+
 	var self = this;
 
-	this.debug              = true;
-	this.container          = null;
-	this.containerObj       = null;
-	this.containerName      = null;
-	this.fileName           = null;
-	this.totalFrames        = null;
-	this.framesLoaded       = 0;
-	this.frameIndex         = 1;
-	this.lastFrameIndex     = 1;
-	this.dragging           = false;
-	this.lastScreenX        = 0;
-	this.inertiaInterval    = null;
-	this.direction          = null;
-	this.frameSpeed         = 0;
-	this.inertiaFrameSpeed  = 0;
-	this.timeInertia        = 0;
-	this.inertiaDuration    = 0;
-	this.imageObjects       = Array();
-	this.RAFrunning         = false;
+	self.debug              = true;
```

Comment:
Actually the use of ```self``` is spoused to be used only when ```this``` is out of scope, like in the method ```inertiaRAF```. But I guess is not a big deal to use ```self``` everywhere instead